CHANGELOG.md

```diff
@@ -106,6 +106,7 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a Ch
 ---
 
 ### New
+* Update custom extractor tutorial content
```

Comment:
This isn't needed as the changelog entry will be taken from the merge commit from this PR 🙂 
```suggestion
```